d2bs/kolbot/libs/common/Misc.js

```diff
@@ -750,6 +791,214 @@ var Item = {
 	}
 };
 
+
+   canEquipMerc: function (item, bodyLoc) {
+        if (item.type !== 4) { // Not an item
+            return false;
+        }
+
+        if (!me.getMerc()) { // dont have merc or he is dead
+            return false;
+        }
+
+        if (!item.getFlag(0x10)) { // Unid item
+            return false;
+        }
+
+        var merc = me.getMerc();
+        var curr = this.getEquippedItemMerc(bodyLoc);
+
+        if (item.getStat(92) > merc.getStat(12) || item.dexreq > merc.getStat(2) - curr.dex || item.strreq > merc.getStat(0) - curr.str) { // Higher requirements
+            return false;
+        }
+
+        return true;
+    },
+
+    // Equips an item and throws away the old equipped item
+    equipMerc: function (item, bodyLoc) {
+        if (!this.canEquipMerc(item, bodyLoc)) {
+            return false;
+        }
+
+        // Already equipped in the right slot
+        if (item.mode === 1 && item.bodylocation === bodyLoc) {
+            return true;
+        }
+
+        var i, cursorItem;
+
+        if (item.location === 7) {
+            if (!Town.openStash()) {
+                return false;
+            }
+        }
+
+        for (i = 0; i < 3; i += 1) {
+            if (item.toCursor()) {
+                var gid = item.gid;
```

Comment:
don't think this is needed since you commented out logitem, consider removing this code and related comments

Reply:
I remove the // commented